lib/code_corps/github/sync/comment/comment.ex

```diff
@@ -48,52 +28,31 @@ defmodule CodeCorps.GitHub.Sync.Comment do
   If the sync fails, it will return an `:error` tuple, where the second element
   is the atom indicating a reason.
   """
-  @spec sync(map) :: outcome
-  def sync(payload) do
-    payload
-    |> operational_multi()
-    |> Repo.transaction
-    |> marshall_result()
+  @spec sync(map, map) :: Multi.t
+  def sync(changes, payload) do
+    sync_multi(changes, payload)
```

Comment:
Seems like there's just an extra nested function here we don't really need right now. Did you plan to add another case/clause here, or are you still planning?